docs/projected-volume.md

```diff
@@ -4,10 +4,15 @@ A [projected volume](https://kubernetes.io/docs/concepts/storage/volumes/#projec
 volume sources into the same directory in the container. With the [proposal for file permission handling in projected
 service account volume](https://github.com/kubernetes/enhancements/pull/1598) enhancement, the projected files have
 the correct permissions set including container user ownership for Linux pods when `RunAsUser` is set in the Pod 
-`SecurityContext`. However, for Windows pods this is not the case when `RunAsUsername` is set, due to differences in 
-the way user accounts are managed in Windows. Windows stores and manages local user and group accounts in a database
-file called Security Account Manager (SAM). This database is not shared between the Windows container host and the
-running containers. This prevents the kubelet from setting correct ownership on the files in the projected volume.
+`SecurityContext`. In Windows pods that have a projected volume and `RunAsUsername` set in the Pod `SecurityContext`,
+the ownership is not enforced due to the way user accounts are managed in Windows. Windows stores and manages local
+user and group accounts in a database file called Security Account Manager (SAM). Each container maintains its own
+instance of the SAM database, to which the host has no visibility into while the container is running. Windows
```

Comment:
```suggestion
user and group accounts in a database file called Security Account Manager (SAM). Each container maintains its own
instance of the SAM database; the host machine has no visibility into the SAM database while the container is running. Windows
```
This sentence sounded awkward to me. Does this suggestion mean the same thing?

Reply:
This could imply the SAM database on the host. So it would be safer to leave this as is.